src/sys/unix/selector/mod.rs

```diff
@@ -18,7 +18,7 @@ mod epoll;
         target_os = "redox",
     )
 ))]
-pub(crate) use self::epoll::{event, Event, Events, Selector};
+pub(crate) use epoll::{event, Event, Events, Selector};
```

Comment:
I would probably have kept the `self::` markers.

Reply:
They're not needed any more since rust edition 2018/2021, so I the code I write now I never use it any more.